tests/foreman/ui/test_host.py

```diff
@@ -161,11 +164,10 @@ def setUpClass(cls):
         # Search if Libvirt compute-resource already exists
         # If so, just update its relevant fields otherwise,
         # Create new compute-resource with 'libvirt' provider.
-        resource_url = u'qemu+tcp://{0}:16509/system'.format(
-            settings.server.hostname
+        resource_url = u'qemu+ssh://root@{0}/system'.format(
+            settings.compute_resources.libvirt_hostname
```

Comment:
ah.. sorry..missed that comment. will update the PR for compute_resource setting. thanks